privval/file.go

```diff
@@ -369,6 +372,12 @@ func (pv *FilePV) signVote(chainID string, vote *tmproto.Vote) error {
 	}
 
 	signBytes := types.VoteSignBytes(chainID, vote)
+	extSignBytes := types.VoteExtensionSignBytes(chainID, vote)
+	// We always sign the vote extension. See below for details.
+	extSig, err := pv.Key.PrivKey.Sign(extSignBytes)
+	if err != nil {
+		return err
+	}
```

Comment:
This looks correct. Can we update the tests in `file_test.go` to ensure that this behaves as expected?

Reply:
Does [this test](https://github.com/tendermint/tendermint/pull/8141/files#diff-8881ac9666ab8aa476a419622ae008801c46bfb7693cc47a807fd7bdf32e12a9R279) suffice?

Reply:
yup!